test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -180,11 +168,11 @@ TEST_F(SynchronizerTest, ValidWhenBlockValidationFailure) {
 
   auto wrapper =
       make_test_subscriber<CallExact>(synchronizer->on_commit_chain(), 1);
-  wrapper.subscribe([&test_block](auto commit) {
+  wrapper.subscribe([test_block](auto commit) {
```

Comment:
const ref?

Reply:
That's just a shared_ptr now so no a big reason to pass it by value.